src/clients/stormGlass.ts

```diff
@@ -108,6 +130,37 @@ export class StormGlass {
       throw new ClientRequestError(err.message);
     }
   }
+
+  public getForecastPointsFromCache(
+    lat: number,
+    lng: number
+  ): ForecastPoint[] | undefined {
+    const forecastPointCache = this.cacheUtil.get<ForecastPoint[]>(
+      `forecast_points_${lat}_${lng}`
```

Comment:
The cache key is used in two places and is a string, what about adding a method like: `getCacheKey(lat, lng): string` that generates the key? This way there will be only one source of truth

Reply:
Nice, I'll implement it!